test/integration/component/test_tags.py

```diff
@@ -2543,3 +2543,248 @@ def test_24_public_IP_tag(self):
             "List tags should return empty response"
         )
         return
+    @attr(tags=["advanced"], required_hardware="false")
```

Comment:
@DaanHoogland  it can be run on simulator.  that is the reason required_hardware is set to false. May I now the concern here